### PR TITLE
[faq](doc) add some FAQ for flink-doris-connector

### PR DESCRIPTION
**Pull request by yagagagaga**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/ecosystem/flink-doris-connector.md [hunk 1]
```diff
@@ -553,3 +553,10 @@ Flink在数据导入时，如果有脏数据，比如字段格式、长度等问
 11. **源表和Doris表应如何对应？**
 使用Flink Connector导入数据时，要注意两个方面，第一是源表的列和类型跟flink sql中的列和类型要对应上；第二个是flink sql中的列和类型要跟doris表的列和类型对应上，具体可以参考上面的"Doris 和 Flink 列类型映射关系"
 
+12. **TApplicationException: get_next failed: out of sequence response: expected 4 but got 3**
+
+如果这是偶发性的出现，可能是当前 connector 版本与 flink 版本存在不兼容，你可以升级 flink 版本或使用较低版本的 connector。
```
Comment (JNSimba) [inline docs/zh-CN/docs/ecosystem/flink-doris-connector.md:558]: Just say that it is enough to upgrade the flink connector version? This is caused by a bug in thrfit concurrency

Comment (yagagagaga) [reply docs/zh-CN/docs/ecosystem/flink-doris-connector.md]: done

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by yz-jayhua — APPROVED** [review]

**Comment by yagagagaga** [comment]
run buildall

**Comment by yagagagaga** [comment]
run buildall

**Comment by yagagagaga** [comment]
run buildall

**Comment by yagagagaga** [comment]
run buildall

**Review by vinlee19 — APPROVED** [review]

**Comment by yagagagaga** [comment]
run buildall

**Comment by yagagagaga** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**